README.md

```diff
@@ -79,6 +79,7 @@ module: {
 
 - __naming__: [bem-naming](https://en.bem.info/toolbox/sdk/bem-naming) overrides
 - __levels__ <Array>: paths to components declarations
+- __set__ <String>: string with name of set
```

Comment:
and It be cool to have link to this docs: https://github.com/bem/bem-sdk/pull/213/files#diff-621cc23937ce7f99fed2fab23dbbbee0R160
but not necessary right now